EpsilonCCS/Inc/TelemetryReporting/TelemetryUtils.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#define CHECKSUM_LENGTH (2)
+#define FRAMING_LENGTH_INCREASE (2)
+#define TERMINATING_BYTE (0x00)
+
+union DataUnion
```

Comment:
You know I thought about that, and put it in the header file because it's more "proper". But you're right, it's not going to be used by anything else, so better to not muddy up the namespace.